libs/template/src/lib/core/utils/make-zone-less.ts

```diff
@@ -0,0 +1,52 @@
+import { apiZonePatched, getGlobalThis } from '@rx-angular/template';
```

Comment:
i am pretty sure those imports should be relative

Reply:
done